### ISSUE TITLE
Add link to the organization/brigades webpage in the brigade column

### ISSUE THREAD

**Issue by k3KAW8Pnf7mkmdSMPHz27** [codeforamerica/brigade-project-index-statusboard#144]
# Is your feature request related to a problem? Please describe.**

Add a link to the organization's webpage if it is known. It should be possible to do it in a similar fashion to the GitHub repository in the project name column.

**Comment by giosce** [comment]
@k3KAW8Pnf7mkmdSMPHz27, this is implemented in the Brigades page.
Would you suggest to do in Projects page too?

**Comment by k3KAW8Pnf7mkmdSMPHz27** [comment]
@giosce yes, I'd consider it a good idea to have on the Projects page as well.
I am a little bit concerned about putting too many links there, and it isn't as relevant as the project link itself. But I don't think clicking on it by accident is an easy mistake to make.
